### ISSUE TITLE
Add IOS Shortcut Action to Create a Folder within a Vault

### ISSUE THREAD

**Issue by ACK-J** [cryptomator/ios#284]
### Please agree to the following

- [X] I have searched [existing issues](https://github.com/cryptomator/ios/issues?q=) for duplicates
- [X] I agree to follow this project's [Code of Conduct](https://github.com/cryptomator/ios/blob/develop/.github/CODE_OF_CONDUCT.md)

### Summary

There is no option to create a folder using an IOS shortcut.

### Motivation

Right now an IOS shortcut automation does not have the option to create a folder within an unlocked vault. When a shortcut is used for automatic backups, the setup of the folder structure needs to be done manually which is a pain.

**Comment by marcos1** [comment]
Yes I too want this feature please add an option to create a folder this way we can sort our photos better
